go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/kubevirt/kubesecondarydns
 
-go 1.19
+go 1.23.0
```

Comment:
can you do 1.23.7 ? so it's aligned with the toolhain ?

Reply:
i dont think we usually do that
even renovate bot doesnt
https://github.com/k8snetworkplumbingwg/kubemacpool/pull/461/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R3
i prefer not please